### PR TITLE
New glider cals

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09231__20220206.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9231,CC_calibration_date,1644105600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09231__20220206.csv:2]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09686__20200907.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9686,CC_calibration_date,1599436800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09686__20200907.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00192__20220309.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+192,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
+192,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00192__20220309.csv:3]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00323__20220221.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+323,CC_calibration_date,1645401600000,sensor calibration date (milliseconds since 1970-01-01)
+323,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00323__20220221.csv:3]: looks good

File: calibration/FLORDM/CGINS-FLORDM-06990__20210809.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+6990,CC_angular_resolution,1.096,
+6990,CC_depolarization_ratio,0.039,
+6990,CC_measurement_wavelength,700.0,
+6990,CC_scattering_angle,140.0,
```
Comment (rtravis-whoi) [inline calibration/FLORDM/CGINS-FLORDM-06990__20210809.csv:5]: looks good

File: calibration/FLORTM/CGINS-FLORTM-03501__20220111.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3501,CC_angular_resolution,1.076,
+3501,CC_depolarization_ratio,0.039,
+3501,CC_measurement_wavelength,700.0,
+3501,CC_scattering_angle,124.0,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-03501__20220111.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01239__20220126.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1239,CC_angular_resolution,1.076,
+1239,CC_depolarization_ratio,0.039,
+1239,CC_measurement_wavelength,700.0,
+1239,CC_scattering_angle,124.0,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01239__20220126.csv:5]: looks good

File: calibration/PARADM/CGINS-PARADM-50213__20151028.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50213,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50213__20151028.csv:2]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09231__20220206.csv (hunk 1 above)
Comment (sbeaulieu) [inline calibration/CTDGVM/CGINS-CTDGVM-09231__20220206.csv:2]: ok

File: calibration/CTDGVM/CGINS-CTDGVM-09686__20200907.csv (hunk 2 above)
Comment (sbeaulieu) [inline calibration/CTDGVM/CGINS-CTDGVM-09686__20200907.csv:2]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00192__20220309.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+192,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (sbeaulieu) [inline calibration/DOSTAM/CGINS-DOSTAM-00192__20220309.csv:2]: ok assuming calibration date DD.MM.YYYY

File: calibration/FLORDM/CGINS-FLORDM-06990__20210809.csv (hunk 5 above)
Comment (sbeaulieu) [inline calibration/FLORDM/CGINS-FLORDM-06990__20210809.csv:5]: looks good

File: calibration/FLORTM/CGINS-FLORTM-03501__20220111.csv (hunk 6 above)
Comment (sbeaulieu) [inline calibration/FLORTM/CGINS-FLORTM-03501__20220111.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01239__20220126.csv (hunk 7 above)
Comment (sbeaulieu) [inline calibration/FLORTO/CGINS-FLORTO-01239__20220126.csv:5]: looks good

File: calibration/PARADM/CGINS-PARADM-50213__20151028.csv (hunk 8 above)
Comment (sbeaulieu) [inline calibration/PARADM/CGINS-PARADM-50213__20151028.csv:2]: confirmed scaling 1 through Slack question re: 2015 calibration

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**